src/index.js

```diff
@@ -1,5 +1,6 @@
 /* global process */
```

Comment:
I think this line is unnecessary

Reply:
Correct, this should take care of it - https://github.com/okonet/lint-staged/blob/80cbfd165911cac122d0216362267266bfa4a910/.eslintrc.json#L5-L10

Reply:
Yeah, already updated. Thanks